grails-app/views/order/addComment.gsp

```diff
@@ -78,5 +78,13 @@
 			</div>
 		</div>
 	</div>
+<script>
+	$(document).ready(function() {
+		const menuConfigValues = $(".menu-config-value").toArray();
+		const stockMovementDirection = ${orderInstance.orderType.isPutawayOrder()} ? 'inbound' : 'purchasing';
+		const foundSection = menuConfigValues.find(it => stockMovementDirection === it.name)
+		applyActiveSection(foundSection)
```

Comment:
Two of the above lines could be moved to the `applyActiveSection` and only the `stockMovementDirection` could be passed instead of `foundSection`. That would reduce the amount of lines copy pasted.